### ISSUE TITLE
Provide a name for downloaded CSV files

### ISSUE THREAD

**Issue by waldoj** [openva/vabusinesses.org#35]
Right now it's `download.csv` which is obviously not good.

Challenge: suggest a name to the browser without forcing a download. That is, if the user agent can native display CSV files, there's no reason to force it to download the file to the desktop.

**Comment by waldoj** [comment]
I'm torn. One cannot propose a filename without forcing a download.

If it's best to force a download, we just need to add this:

```php
header('Content-Disposition: attachment; filename="whatever.csv"');
```

It's probably best to allow setting `$this->filename` so that we can pass a filename—it might be the search term that was being used (and is now being exported), or it might be a locality name.

**Comment by waldoj** [comment]
Closed with cb41e4c16f069e203eb312b580ad075844b193d8.
